configs/todc-bootstrap-v3.json

```diff
@@ -1,6 +1,7 @@
 {
-  "index_name": "todc-bootstrap",
+  "index_name": "todc-bootstrap-v3",
   "start_urls": [
+    "https://todc.github.io/todc-bootstrap/",
```

Comment:
Please give only one protocole. Otherwise you will have duplicated data

Reply:
Okay. FYI - I was following what is in the bootstrap-v3 config.

Reply:
Thanks. It is possible to do it as long as you have canonical url